src/test/java/org/opensearch/ad/ml/ModelManagerTests.java

```diff
@@ -266,6 +277,9 @@ public void setup() {
         detectorId = "detectorId";
         rcfModelId = "detectorId_model_rcf_1";
         thresholdModelId = "detectorId_model_threshold";
+
+        when(this.modelState.getModel()).thenReturn(this.entityModel);
+        when(this.entityModel.getErcf()).thenReturn(Optional.of(this.ercf));
```

Comment:
This would make it hard to test the else branch of entityModel.getErcf().isPresent() == false.  Can we move it to somewhere else?

Reply:
it shouldn't not affect new test cases since new test cases can always override this behavior so this is just a default for convenience.